ndk-macro/src/helper.rs

```diff
@@ -42,22 +43,21 @@ impl Parse for AttributeArgs {
 use proc_macro_crate::crate_name;
 
 #[cfg(test)]
-fn crate_name(name: &str) -> Result<String> {
-    Ok(name.replace('-', "_"))
+fn crate_name(name: &str) -> Result<FoundCrate> {
+    Ok(FoundCrate::Name(name.replace('-', "_")))
 }
 
 pub fn crate_path(name: &str, overridden_path: &Option<Path>) -> Path {
     // try to use overridden crate path
     overridden_path.clone().unwrap_or_else(|| {
-        // the binding to hold string from `crate_name` fn
-        let mut detected_name = None;
         Ident::new(
             // try to determine crate name from Cargo.toml
             crate_name(name)
                 .ok()
-                .map(|name| {
-                    detected_name = Some(name);
-                    detected_name.as_ref().unwrap().as_str()
+                .as_ref()
+                .map(|name| match name {
+                    FoundCrate::Itself => todo!(),
```

Comment:
This should probably be `"ndk_macro"`?

Reply:
looks like it

Reply:
Let's get that in then and merge this :)